speechbrain/lobes/models/huggingface_wav2vec.py

```diff
@@ -457,7 +457,7 @@ def forward(self, wav, wav_lens):
         torch_mask_time_indices = torch.tensor(
             mask_time_indices, device=wav.device, dtype=torch.long,
         )
-        padding_mask = self.make_masks(wav, wav_len=wav_lens)
+        padding_mask = self.make_padding_masks(wav, wav_len=wav_lens)
```

Comment:
why? It seems to be the same for me.. not?

Reply:
different classes but the same function has a different name in each class